recipes/cpp_project_framework/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package ${CONAN_LIBS})
```

Comment:
Why is the compiler requirement not used here https://github.com/conan-io/conan-center-index/pull/11483/files#diff-a4d76a1b3769e1ec67e99a76048eb4dc0691ba8610016de787b2d16207c8899bR30-R31 ?

Reply:
added c++ standard requirement